### ISSUE TITLE
Return History

### ISSUE THREAD

**Issue by TKassis** [keras-team/autokeras#1423]
### Feature Description
In Keras it's possible to plot validation loss by accessing it through model.history, but Auto-Keras does not return an object containing this when I call .fit. What's the best way to access historical metrics for the best model?

**Comment by gousemd73** [comment]
Yes, It would be very nice if can able plot the losses and accuracies that we recorded during the training.

**Comment by haifeng-jin** [comment]
May be you can try `auto_model.tuner.results_summary()`? We will have a better solution in the next few months by improving Keras Tuner.

**Comment by hanzigs** [comment]
Yes, I am waiting for this as well, Thanks

**Comment by pa-blan** [comment]
@haifeng-jin You can already use Kerastuner to create the history of the final model, why not in autokeras?

**Comment by SuiAn-97** [comment]
Waiting for this.

**Comment by lwq-star** [comment]
What's the way to access historical metrics? I still receive this error.
AttributeError: 'StructuredDataRegressor' object has no attribute 'history'
`reg = ak.StructuredDataRegressor(overwrite=True,max_trials=3,
                                     metrics=[ta.metrics.RSquare(),
                                              tf.keras.metrics.RootMeanSquaredError(),"mse"])
 reg.fit(x, y, epochs=10)
 history = reg.history`

**Comment by lwq-star** [comment]
What's the way to access historical metrics? I still receive this error.
AttributeError: 'StructuredDataRegressor' object has no attribute 'history'

```
reg = ak.StructuredDataRegressor(overwrite=True,max_trials=3, metrics=[ta.metrics.RSquare(), 
                                                                                                                      tf.keras.metrics.RootMeanSquaredError(),"mse"]) 
reg.fit(x, y, epochs=10) 
history = reg.history
```

**Comment by BenUCL** [comment]
> 

This returns AttributeError: 'Functional' object has no attribute 'tuner'
